flepimop/gempyor_pkg/src/gempyor/calibrate.py

```diff
@@ -113,18 +137,37 @@
 # @profile_options
 # @profile()
 def calibrate(
-    config_filepath,
-    project_path,
-    nwalkers,
-    niter,
-    nsamples,
-    nthin,
-    ncpu,
-    input_run_id,
-    prefix,
-    resume,
-    resume_location,
-):
+    config_filepath: str,
+    project_path: str,
+    nwalkers: int,
+    niter: int,
+    nsamples: int,
+    nthin: int,
+    ncpu: int,
+    input_run_id: int,
+    prefix: str,
+    resume: bool,
+    resume_location: str,
```

Comment:
I think some of these are missing a union with `None`, `resume_location` for example.